CHANGELOG.md

```diff
@@ -7,10 +7,13 @@ This is the log of notable changes to the Expo client that are developer-facing.
 ### 📚 3rd party library updates
 
 ### 🛠 Breaking changes
+- `Google.logInAsync()` now accepts a `redirectUrl` value for apps running in the Expo Client. Previously, it would ignore this, so if you are passing a value, make sure to [follow the guidelines](https://docs.expo.io/versions/latest/sdk/google/#loginasync) ([#4904](https://github.com/expo/expo/pull/4904) by [@cruzach](https://github.com/cruzach))
```

Comment:
not sure why this wasn't published with `expo-google-app-auth` v6.0.0 🤷‍♂ 
but lgtm!